bitnami/dataplatform-bp1/values.yaml

```diff
@@ -27,6 +39,24 @@ commonAnnotations: {}
 ## @param extraDeploy Array of extra objects to deploy with the release
 ##
 extraDeploy: []
+## @param clusterDomain Kubernetes cluster domain name
+##
+clusterDomain: cluster.local
+
+## Enable diagnostic mode in the deployment(s)/statefulset(s)
+##
+diagnosticMode:
```

Comment:
Could you add some information in NOTES.txt about the `diagnosticMode`?

Reply:
I had to drop diagnosticMode because dataplatform-bp1 uses `scratch` containers